pom.xml

```diff
@@ -29,15 +29,15 @@
 		<project.jdk.version>20</project.jdk.version>
 
 		<!-- Group IDs of jars that need to stay on the class path for now -->
-		<!-- Once hypfvieh, swiesend, purejava and integrations-linux have module-info, remove them-->
-		<nonModularGroupIds>org.ow2.asm,org.apache.jackrabbit,org.apache.httpcomponents,de.swiesend,org.purejava,com.github.hypfvieh</nonModularGroupIds>
+		<!-- Once swiesend has module-info, remove them-->
+		<nonModularGroupIds>org.ow2.asm,org.apache.jackrabbit,org.apache.httpcomponents</nonModularGroupIds>
 
 		<!-- cryptomator dependencies -->
 		<cryptomator.cryptofs.version>2.6.5</cryptomator.cryptofs.version>
-		<cryptomator.integrations.version>1.2.0</cryptomator.integrations.version>
+		<cryptomator.integrations.version>1.3.0-beta1</cryptomator.integrations.version>
```

Comment:
will release a stable version today

Reply:
Done in https://github.com/cryptomator/cryptomator/pull/2885/commits/238aaeb5a0108a88443ff16e1e4ff6da0b27288e